README.md

```diff
@@ -92,7 +103,59 @@ Then configure your Notification Content target Info.plist to reflect the catego
 
 Also, If you plan on downloading non-SSL urls please be sure to enable `App Transport Security Settings -> Allow Arbitrary Loads -> true` in your plist.  [See plist example here](https://github.com/CleverTap/CTNotificationContent/blob/master/Example/NotificationContent/Info.plist).  
 
-### Configure your APNS payload
+# Dashboard Usage
+
+[(Back to top)](#table-of-contents)
+While creating a Push Notification campaign on CleverTap, just follow the steps below -
+
+1. On the "WHAT" section pass the desired required values in the "title" and "message" fields (NOTE: These are iOS alert title and body).
+![Dashboard alert](https://github.com/CleverTap/CTNotificationContent/blob/master/images/dashboard_alert.png)
+2. Click on "Advanced" and then click on "Rich Media" and select Single or Carousel template.
+![Dashboard Rich Media](https://github.com/CleverTap/CTNotificationContent/blob/master/images/dashboard_richMedia.png)
+3. For adding custom key-value pair, use the [template Keys](#template-keys) into one JSON object and use the `pt_json` key to fill in the values.
+![Dashboard Custom Key value](https://github.com/CleverTap/CTNotificationContent/blob/master/images/dashboard_customKeyValue.png)
+4. Send a test push and schedule!
+
+# Template Types
+
+[(Back to top)](#table-of-contents)
+
+## Rich Media
+### Single Media
+Single media is for basic view with single image.
+![Single Media](https://github.com/CleverTap/CTNotificationContent/blob/master/images/SingleMedia.png)
+
+### Content Slider
+Content Slider is for image slideshow view where user can add multiple images with different captions, sub-captions, and actions.
+<img src="https://github.com/CleverTap/CTNotificationContent/blob/master/images/ContentSlider.gif" alt="Content slider" width="450" height="800"/>
+
+## Custom key-value pair
+
+### Basic Template
+Basic Template is the basic push notification received on apps where user can also update text colour, background colour.
+![Custom Basic template](https://github.com/CleverTap/CTNotificationContent/blob/master/images/CustomBasicTemplate.png)
+
+### Auto Carousel Template
+Auto carousel is an automatic revolving carousel push notification where user can also update text colour, background colour.
+<img src="https://github.com/CleverTap/CTNotificationContent/blob/master/images/CustomAutoCarousel.gif" alt="Auto carousel" width="450" height="800"/>
+
+### Manual Carousel Template
+This is the manual version of the carousel. The user can navigate to the next/previous image by clicking on the next/previous buttons.
```

Comment:
@nishant-clevertap Instead of "Next/Previous buttons" it should be "Next/Back" Buttons as per our category